django/contrib/auth/models.py

```diff
@@ -145,7 +145,6 @@ def _create_user(self, username, email, password, **extra_fields):
         if not username:
             raise ValueError('The given username must be set')
         email = self.normalize_email(email)
```

Comment:
👍  Yes, thanks a lot. Somehow I forgot to delete that line after several double checks 😞